tutorbackup/templates/backup/build/backup/restore_services.py

```diff
@@ -64,6 +64,7 @@ def restore_mongodb():
     logger.info(
         f"Restoring MongoDB databases on {host}:{port} from {dump_dir}")
     cmd = ("mongorestore "
+           f"{ENV['MONGORESTORE_ADDITIONAL_OPTIONS']} "
            "--stopOnError --drop "
            f"--host={host} --port={port} "
            f"{dump_dir}"
```

Comment:
I don't have a good explanation why, but to me it "feels" more correct if we inject the "additional" options at the end (i.e. just before `f"{dump_dir}"`).